matthew-bavosa.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html> <!--added doctype-->
+<html>
+  <head>
+    <title>CSS Timeline</title>
+      <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <link rel="stylesheet" href="css/matthew-bavosa.css">
+  </head>
+  <body>
+    <h1>Brief History of CSS</h1><!--fixed closing tag-->
+        <ul>
+          <li id="happy-matt"><div class="bold-and-italic">1994</div>: The first work on style sheets started at CERN.</li>
+          <li>December <div class="bold-and-italic">1996</div>: CSS was proposed as W3C standard:</li>
```

Comment:
How about we use a `<span>` instead of a `<div>` here?

`<span>` is an inline element, which means we don’t need `display: inline;` … makes sense?